src/main/java/org/opentripplanner/util/OTPFeature.java

```diff
@@ -19,6 +19,7 @@
     APIUpdaterStatus(true),
     OptimizeTransfers(true),
     GuaranteedTransfers(true),
+    ParallelRouting(false),
```

Comment:
I think this should be enabled by default, disabling only for debugging,

Reply:
Enabling this _may_ bring out race conditions -- for example the `TurnRestriction` modification isn't thread-safe (`Graph#addTurnRestriction`, `Graph#removeTurnRestriction`, `Graph#getTurnRestrictions`), which is called when creating a `RoutingContext` / `StreetVertexIndex#getVerticesForLocation()`. Due to the above parallelization 2-3 `RoutingContext`s may be created at the same time in different threads.

Reply:
I've updated the default to be `true`.